clustered/client/src/test/java/org/ehcache/clustered/client/docs/GettingStarted.java

```diff
@@ -114,4 +116,28 @@ public void clusteredCacheManagerWithDynamicallyAddedCacheExample() throws Excep
     // end::clusteredCacheManagerWithDynamicallyAddedCacheExample[]
   }
 
+  @Test
+  public void explicitConsistencyConfiguration() throws Exception {
+    final CacheManagerBuilder<PersistentCacheManager> clusteredCacheManagerBuilder
+            = CacheManagerBuilder.newCacheManagerBuilder()
+            .with(ClusteringServiceConfigurationBuilder.cluster(URI.create("terracotta://localhost:9510/my-application?auto-create"))
+                    .defaultServerResource("primary-server-resource")
+                    .resourcePool("resource-pool-a", 128, MemoryUnit.B));
```

Comment:
Seems overkill but ready to hear arguments. We should however pick a default when nothing is selected at the cache level.